src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+
+- Opening a door! Minimal amount required.
+- Speech when the radio process is inactive. A somewhat substantial amount when surviving off a single or two AI servers, but not extremely hard hitting. Binary channel is always free however.
+- Left-click or alt-click on vents to unlock them, for cheap
+- **TODO**. Ideas: sensor sweep? Mass applying (un)bolting to everything an APC’s connected to? Atmospherics reset action that can be used from the action bar/alt-click radial menu and applied to an atmos alarm to reset it to default? Clicking vents to unlock them?
+- Consoles! They aren’t free and cut a decent chunk, to encourage the AI player to choose a Compute option, but in a pinch this should work too. It also gets paid by the player having to manually locate a console and being unable to check on things elsewhere but the vicinity of the console.
+
+### Sources
+
+- AI specialized machinery, contributing a significant amount of processing capability to the AI.
+  - The downside to this is that it requires someone physical to build it, and needs a decent amount of resources
+  - Such machinery should be difficult to destroy or even move. It stays where it stays, unless the situation requires it and someone puts enough effort to move it; or break it.
+  - Generates heat up to a throttle cutoff, and works better when cooled. When not in an optimal environment, provides less computing power, but still comparable to servers.
+  - Backup power? Camera built in?
+- Crew devices; computers, machines, servers
+  - These don’t contribute as much **Compute** as they are not specialized for AI use. Servers do provide more **Compute** than just consoles, and provide **Bandwidth** in amounts comparable or identical to dedicated machines..
+  - Locks out the device for crew use, rendering it inoperable. This acts as a conflict point with the crew. Computers don’t work, machines either. Servers don’t provide normal functionality or do so in a very limited way.
+    - E.g. crew monitoring server could give invalid/partial data with a warning, or just not work at all. Same for telecom servers.
+  - Non-antag AI can be flushed out from the device via a near-instant crew exposed action. This includes a subverted AI. Could be an UI that shows “This device is in use by AI \<name\>. \<Override (button)\>.”
+  - Such shooing away should keep the AI from taking the device over for at least some time, at least a minute. It should not be able to just annoy the crew to death even when compromised - it’s still shackled.
+  - Antagonist AI, “Malf”, should have an option or be able to spend some resources to completely lock the crew out. Will require more detailed balancing later.
+  - For “Malf”, the timeout should be reduced or bypassable. Again, might need some balance tweaking whether it’s just a pop up “you sure?” or a resource sink.
```

Comment:
Good point. Is Malf AI taking over pointless, little machines, one by one, a source of resources, or will it require Bits/s? IMO it should the latter, as this might provide a way for Malf AI to claw some much-needed processing power in its final moments (when it's very obvious that it's Malf). The crew could just shoot the vending machines or snip the AI wire to cut out the gained processing power. Could be something cool to see engineering do in an effort to combat an active Malf.